games-puzzle/blockout/blockout-2.5.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit desktop
+
+DESCRIPTION="BlockOut II is an adaptation of the original Blockout DOS game"
+
+HOMEPAGE="http://www.blockout.net/blockout2"
+SRC_URI="
+	mirror://sourceforge/blockout/bl25-src.tar.gz
+	mirror://sourceforge/blockout/bl25-linux-x86.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	media-libs/libsdl
+	media-libs/sdl-mixer
+	virtual/glu
+	virtual/opengl
+	x11-libs/libXext"
+
+DEPEND="${RDEPEND}"
+
+src_unpack() {
+	unpack ${A}
+	mv "BL_SRC" ${P}
+}
```

Comment:
No need for a `src_unpack` function, define `S="${WORKDIR}"/BL_SRC` instead

Reply:
Removed